container-assets/container-entrypoint

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+set -e
+
+CA_CERT_PATH=$CA_CERT_PATH
+CA_KEY_PATH=$CA_KEY_PATH
+KEYSTORE_PASS=$KEYSTORE_PASS
```

Comment:
We will probably need to add these as env vars here? https://github.com/ManageIQ/manageiq-pods/blob/master/manageiq-operator/api/v1alpha1/helpers/miq-components/kafka.go#L229